pkger/models.go

```diff
@@ -767,6 +768,22 @@ func (c chart) properties() influxdb.ViewProperties {
 			Note:              c.Note,
 			ShowNoteWhenEmpty: c.NoteOnEmpty,
 		}
+	case chartKindScatter:
+		return influxdb.ScatterViewProperties{
+			Type:              influxdb.ViewPropertyTypeScatter,
+			Queries:           c.Queries.influxDashQueries(),
+			ViewColors:        c.Colors.strings(),
+			XColumn:           c.XCol,
+			YColumn:           c.YCol,
+			XAxisLabel:        c.Axes.influxAxes()["x"].Label,
```

Comment:
you may want to run the `c.Axes.influxaxes()` call on before the return statement, then reuse its contents here instead of regenerating the influxaxes 6 times